### DISCUSSION TITLE
Application as a student being rejected

### DISCUSSION THREAD

**Discussion by akamikado** [community/community#61060] [GitHub Education]
### Hi everyone,

I belong to BITS Pilani, Hyderabad Campus. When I apply using my college email, my college is automatically chosen as BITS Pilani, Pilani campus and it says that my location does not match the campus location. Many students of my campus(Hyderabad campus) already have Github student pack so I don't know why I'm facing a problem. I'm not asked for giving additional proof. How should I proceed with this?

**Comment by Someet-Git** [comment]
I am also having the same issue like it is saying because of the location didn't match or like u r not currently in your campus, please help us with this. 
@samus-aran can you tell us how to proceed with this.
